src/main/resources/hudson/plugins/git/GitSCM/global.jelly

```diff
@@ -11,6 +11,9 @@
     <f:entry title="${%Create new accounts based on author/committer's email}" field="createAccountBasedOnEmail">
            <f:checkbox name="createAccountBasedOnEmail" checked="${descriptor.createAccountBasedOnEmail}"/>
     </f:entry>
+    <f:entry title="${%Show the entire commit summary in changes}" field="truncateSummary">
```

Comment:
The variable 'truncateSummary' in this context doesn't mean truncate the summary.  When it is true, it means "show the entire commit summary in changes".  When it is false, it means "Do not show the entire commit summary in changes".

Could we change this variable to be named the way it is used?  For example, use the name `showEntireCommitSummaryInChanges` here as is used in other parts of the code?